### PR TITLE
Make accordion in select attribute values great again

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | -----
| Branch?         | 1.1
| Bug fix?        | yes
| New feature?    | no
| BC breaks?      | no
| Deprecations?   | no
| Related tickets | fixes https://github.com/Sylius/Sylius/issues/8900
| License         | MIT

For sure it's not the best solution from the js point of view, but it works :)
The problem was that newly added select attribute values has translation accordion inside which needs to be triggered to make them work.

### CODE REVIEW

File: src/Sylius/Bundle/AdminBundle/Resources/private/js/sylius-select-attribute.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+(function ($) {
+    'use strict';
+
+    $.fn.extend({
+        handleSelectAttributeTranslatableValues: function () {
+            var element = $(this);
+
+            element.find('[data-form-collection="add"]').on('click', function() {
+                setTimeout(function() {
```
Comment (stefandoorn) [inline src/Sylius/Bundle/AdminBundle/Resources/private/js/sylius-select-attribute.js]: What is the benefit of waiting 100ms? Or why doesn't it work without I mean..

Comment (Zales0123) [reply src/Sylius/Bundle/AdminBundle/Resources/private/js/sylius-select-attribute.js]: If we listen on "click" of the **Add** button, it takes some miliseconds to add this value, triggering `accordion()` immediately don't catch the latest element. As I said, I'm not a JS-guy at all, I just know that it helps 😄 I'll appreciate any better solution.

Comment (stefandoorn) [reply src/Sylius/Bundle/AdminBundle/Resources/private/js/sylius-select-attribute.js]: Probably listening to the adding of the element is 'more clean' (https://stackoverflow.com/questions/18746825/adding-an-event-to-catch-when-a-list-item-is-added-to-a-select-jquery), but this also does the job probably :-D

Comment (Zales0123) [reply src/Sylius/Bundle/AdminBundle/Resources/private/js/sylius-select-attribute.js]: @stefandoorn this `DOMNodeInserted` trick did the job! Thanks 😄

**Pull request merged**